xds/internal/xdsclient/transport/transport.go

```diff
@@ -379,7 +389,7 @@ func (t *Transport) send(ctx context.Context) {
 				continue
 			}
 			if err := t.sendAggregatedDiscoveryServiceRequest(stream, resources, url, version, nonce, nackErr); err != nil {
-				t.logger.Warningf("ADS request for {resources: %q, url: %v, version: %q, nonce: %q} failed: %v", resources, url, version, nonce, err)
+				t.logger.Warningf("Sending ADS request for resources: %q, url: %q, version: %q, nonce: %q failed: %v", resources, url, version, nonce, err)
```

Comment:
I had a comment above for the `err` returned from `sendAggregatedDiscoveryServiceRequest`. Considering that, should we add a DebugF log here which logs `pretty.ToJSON(resp)`?

Reply:
Made the change suggested in the other comment. I feel the only information useful from the whole message are the fields which are printed separately here. I feel this is more readable than printing the whole message with a `pretty.JSON()`. Let me know if you feel otherwise.

Reply:
agreed.